### DISCUSSION TITLE
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/message/saver.rs:220:17

### DISCUSSION THREAD

**Discussion by FujiSanJonathan** [proshunsuke/colmsg#78] [Q&A]
<img width="563" alt="截圖 2022-09-05 下午9 32 48" src="https://user-images.githubusercontent.com/112770255/188461457-8aa55bc1-dbc4-456c-8007-1309b4b002d9.png">
How can fix it? I don't know what going on

**Comment by proshunsuke** [comment]
@FujiSanJonathan  If you delete the 影山優佳 folder and re-run the program, do you still get the same problem?
